test/lib/generators/associated_generator_test.rb

```diff
@@ -33,11 +33,11 @@ class Organization::Seats < ActiveRecord::AssociatedObject
   end
 
   test "connects record" do
-    run_generator
+    run_generator ["Organization::SeatsManager"]
 
     assert_file "app/models/organization.rb", <<~RUBY
       class Organization
-        has_object :seats
+        has_object :seats_manager
       end
     RUBY
```

Comment:
We still want the original test too to check that we keep the pluralization. Can you add a separate test for this?

If you want to, we probably ought to test with `run_generator ["organization/seats_manager"]` too, but I can do that separately.

Reply:
Totally fair.

Just pushed another update. Added two new tests.

To make it work with lower_snake_case paths I needed to make a small adjustment to the methods that compute the class names.